package.json

```diff
@@ -66,7 +66,8 @@
     "tsdx": "^0.14.1",
     "tslib": "^2.3.1",
     "typechain": "^7.0.1",
-    "typescript": "^4.6.2"
+    "typescript": "^4.6.2",
```

Comment:
I don't think adding `winston` to the `devDependencies` would slow the Vercel API down but i'm not positive

Reply:
Are you just adding winston to try things out?

Reply:
Agree @nicholaspai. devDependencies don't get bundled with the package.